### DISCUSSION TITLE
官网编辑器申请内测无法打开

### DISCUSSION THREAD

**Discussion by ArvinC** [galacean/discussion#85] [Engine Toolkit] [answered]
官网编辑器申请内测无法打开，显示无法访问该网站

**Comment by eyworldwide** [comment, marked as answer]
@ArvinC https://survey.alipay.com/apps/zhiliao/o1u1SSHk1 是这个链接打不开吗？

**Reply by ArvinC** [reply]
是的

**Reply by eyworldwide** [reply]
有点奇怪，这个问卷是公网都能访问的。请留下你的邮箱（用来注册帐号），我帮你加一下，到时候请关注内测邀请邮件。

**Reply by ArvinC** [reply]
arvin@caojiawei.com
十分感谢您的支持

**Reply by eyworldwide** [reply]
邀请码已发送

**Reply by Developer-Tang** [reply]
我也出现一样的问题
![image](https://github.com/galacean/discussion/assets/56823520/31399c16-a166-4f49-9456-9a1341dc6778)

**Reply by cptbtptpbcptdtptp** [reply]
你好 麻烦留一下邮箱，我们添加一下。

**Reply by Developer-Tang** [reply]
tang_0416@126.com，麻烦了
